pages/common/adb-logcat.md

```diff
@@ -0,0 +1,20 @@
+# logcat
```

Comment:
```suggestion
# adb-logcat
```
Even though it is an extension of `logcat` , I think mentioning `adb-logcat` would be better